src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -166,9 +166,6 @@ private static bool AddSimpleName(SimpleNameSyntax simpleName, List<string> part
         return true;
     }
 
-    public static bool IsAnyLiteralExpression(this ExpressionSyntax expression)
-        => expression is LiteralExpressionSyntax;
```

Comment:
pointless helper.  removed as i would prefer people make the direct and clear type checks.